cassandra-dac/src/main/java/org/sunbird/cassandra/CassandraOperation.java

```diff
@@ -60,6 +60,15 @@
   public void deleteRecord(
       String keyspaceName, String tableName, Map<String, String> compositeKeyMap);
 
+  /**
+   * @desc This method is used to delete record in cassandra db by their primary key(identifier)
+   * @param keyspaceName Keyspace name
+   * @param tableName Table name
+   * @param identifierList List of Primary Keys
+   * @return Response Response
```

Comment:
Return type is boolean not Response.

@response True if delete operation is successful. Otherwise, return false.

Reply:
amit told me to put boolean, as it will help us in detect fail in delete operation